### PR TITLE
Removes clear button when the input is disabled

### PR DESCRIPTION
**Pull request by tomrndom**
image

ref: https://tipit.avaza.com/task#!sortby=DateUpdatedDesc&task=3210593

Signed-off-by: Tomás Castillo <tcastilloboireau@gmail.com>

### CODE REVIEW

**Review by gcutrini — COMMENTED** [review]
LGTM

**Review by smarcet — CHANGES_REQUESTED** [review]
@tomrndom please review comments

File: src/components/inputs/registration-company-input.js [hunk 1]
```diff
@@ -16,12 +16,13 @@ import PropTypes from 'prop-types'
 import AsyncSelect from 'react-select/lib/Async';
 import { queryRegistrationCompanies } from '../../utils/query-actions';
 
-const RegistrationCompanyInput = ({ error, value, onChange, id, multi, isMulti, className, summitId, onError, inputPlaceholder, DDLPlaceholder, ...rest }) => {
+const RegistrationCompanyInput = ({ error, value, onChange, id, multi, isMulti, disabled, isDisabled, className, summitId, onError, inputPlaceholder, DDLPlaceholder, ...rest }) => {
```
[the remark this answers is not in the document]
Comment (smarcet) [reply src/components/inputs/registration-company-input.js:19]: and why use 2 flags instead of one ?

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**